.github/workflows/create-daily-docs-sync-pr.yaml

```
@@ -56,19 +63,26 @@ jobs:
         run: |
           .github-workflow/scripts/load-translation-bot-config.sh translation-bot.json
 
-      - name: Prepare pull request content
-        id: prepare-pr
+      - name: Check if sync branch already exists
+        id: check-sync-branch
         if: ${{ steps.bot-config.outputs.bot_disabled == 'false' }}
         run: |
-          .github-workflow/scripts/merge-conflicts.sh "$BOT_USERNAME" "$BOT_EMAIL"
+          sync_branch="sync-$(git describe --tags --always "english/develop")"
+          .github-workflow/scripts/check-remote-branch.sh "$sync_branch"
```

So, based in my other comment, this can be:

```suggestion
          branch_exists=$(./scripts/check-remote-branch.sh "$sync_branch")
          echo "::set-output name=branch_exists::$branch_exists"
```

---

This can also be ignored based on https://github.com/solidity-docs/.github/pull/41#discussion_r1051995063

---

https://github.com/solidity-docs/.github/pull/40#issuecomment-1359929125